libs/GameKit/include/SMAutonomousGame.h

```diff
@@ -0,0 +1,41 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "Game.h"
+#include "boost/sml.hpp"
+
+namespace leka::game {
+
+namespace sm::event {
+	struct start_game {
+	};
+	struct stop_game {
+	};
+
+}	// namespace sm::event
+
+namespace sm::state {
+
+	inline auto idle		 = boost::sml::state<class idle>;
+	inline auto playing_game = boost::sml::state<class playing_game>;
+
+}	// namespace sm::state
+
+struct SMAutonomousGame {
```

Comment:
```suggestion
struct AutonomousGamesStateMachine {
```

ou alors mettre dans un namespace `autonomous_games`

Reply:
et changer le nom du fichier aussi

Reply:
ou alors GameKitStateMachine si la logique est dans game kit